connect_ext_ppr/events.py

```diff
@@ -42,7 +43,7 @@ def handle_listing_processing(self, request):
         if request['status'] == 'listed':
```

Comment:
As we are not doing anything with 'unlisted' event, it's better not to subscribe to it at all.

Same with 'uninstalled' event of 'installation_status_change'